Index: PZ_16/123.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from figures import circle_perimeter\n\nprint(circle_perimeter(4))\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/PZ_16/123.py b/PZ_16/123.py
--- a/PZ_16/123.py	(revision 2b14bd5e17c11182115184869b6e591c5ad2abe5)
+++ b/PZ_16/123.py	(date 1677674006053)
@@ -1,3 +1,4 @@
 from figures import circle_perimeter
 
 print(circle_perimeter(4))
+# тебе говорили из проекта выходить
\ No newline at end of file
